doc/integrator/print.rst

```diff
@@ -82,3 +82,9 @@ For this, create a makefile with:
 
    BUILD_RULES = test-packages print
    TEST_PACKAGES = main print
+
+Notes about the scales
+----------------------
+
+The web map and the print did not use the same scales, then you configure the min/max scale denom you
+should also test it in the print to be sure that is works as expected.
```

Comment:
If I understand correctly:

"The web map generation and the map printing do not use the same scales. Therefore, if you change the configuration of min/max scaling, you should also test map printing to be sure that it also works as expected."

Reply:
What the web map **generation**?

Reply:
To distinguish in the processes,
- printing (a map)
- generating (a map) for display in a web browser
These processes use different scaling configurations / mechanisms.

(my understanding - feel free to adapt)